compiler-core/src/docs.rs

```diff
@@ -25,7 +25,16 @@ pub fn generate_html(
     analysed: &[Module],
     docs_pages: &[DocsPage],
 ) -> Vec<OutputFile> {
-    let modules = analysed.iter().filter(|module| !module.is_test());
+    let hidden_modules = config.specify_hidden_modules(
+        analysed
+            .iter()
+            .map(|module| module.name.clone())
+            .collect_vec(),
+    );
+    let modules = analysed
+        .iter()
+        .filter(|module| !module.is_test() && !hidden_modules.contains(&module.name));
```

Comment:
Something like this may be nicer

```rust
let modules = analysed
    .iter()
    .filter(|module| !module.is_test())
    .filter(|module| !config.is_hidden_module(module))